composer/json_schemas/generate_json_schemas.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2022 MosaicML Composer authors
```

Comment:
nit: can you add a shebang? `#!/usr/bin/env python3` should suffice

Reply:
is there a reason why? It looks like no other file has this?